UI/Selenium/UI.csproj

```diff
@@ -27,6 +27,8 @@
     <PackageReference Include="Microsoft.TestPlatform.TestHost" Version="17.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="NLog" Version="4.7.13" />
+    <PackageReference Include="NLog.Extensions.Logging" Version="1.7.4" />
```

Comment:
Are all the NLOG package versions correct?  You might expect them all to be the same version,